lib/utils/transformOptionsFromPath.js

```diff
@@ -0,0 +1,32 @@
+const transformOptionsFromPath = (options, relPathFromRoot) => {
```

Comment:
Probably would be a good idea to add jsdoc since this is a public function. Also, any particular reason why this is defined as a constant arrow function? In other parts of objection.js I see exported functions defined simply as "function xyz (args)"

Reply:
I agree, the same style could be used for these for consistency

Reply:
All right, got some different habits :) I'll fix that. Perhaps directly export the function?